README.md

```diff
@@ -87,6 +87,7 @@ Please also check the documentation of [Language Support for Java by Red Hat](ht
 - `java.debug.settings.showStaticVariables`: show static variables in "Variables" viewlet, defaults to `true`.
 - `java.debug.settings.showQualifiedNames`: show fully qualified class names in "Variables" viewlet, defaults to `false`.
 - `java.debug.settings.maxStringLength`: the maximum length of string displayed in "Variables" or "Debug Console" viewlet, the string longer than this length will be trimmed, defaults to `0` which means no trim is performed.
+- `java.debug.settings.enableHotCodeReplace`: enable hot code replace for Java code. Make sure the auto build is not disabled for [VSCode Java](https://github.com/redhat-developer/vscode-java). See the [wiki page](https://github.com/Microsoft/vscode-java-debug/wiki/Hot-Code-Replace) for more information about usages and limitations.
```

Comment:
`VSCode Java` points to vscode-java marketplace?

Reply:
defaults to `false`.

Reply:
Not sure. I intent to point the the document for the options of the auto build. But there is no specific links.

Reply:
This markdown file resides in GitHub, so pointing to GitHub feel natural.